lnd.go

```diff
@@ -200,9 +200,9 @@ func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan <-chan struct{}) error {
 	}()
 
 	// Show version at startup.
-	ltndLog.Infof("Version: %s commit=%s, build=%s, logging=%s",
+	ltndLog.Infof("Version: %s commit=%s, build=%s, logging=%s debuglevel=%s",
```

Comment:
Please make sure to follow the contribution guidelines for commit messages.
Also the short commit summary should be something like this: 
`lnd: show debug level at startup`
 (We usually add the module name to the commit).

Reply:
Also we should probably add a comma before the `debuglevel` to follow the existing pattern.

Reply:
Are you sure we need a comma, there is not comma between the Version  and commit ? I assumed that logging and debuglevel are similar.

Reply:
Added comma, changed the commit message (took this as an example:
https://github.com/lightningnetwork/lnd/pull/4649)